cluster/addons/dashboard/dashboard-controller.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: extensions/v1beta1
+apiVersion: apps/v1beta2
```

Comment:
why beta api instead of v1 api?

Reply:
Is `v1` API available since kubernetes v1.8? We can change it here since it would work with 1.9 kubernetes but in Dashboard repository we are using this version. If it is available since 1.8 then we will change it.

Reply:
no, apps/v1 is only available in 1.9

Reply:
So we will change it here and keep it as is on Dashboard repo.

Reply:
Done.